src/libraries/Microsoft.PowerFx.Core/Functions/Blob/StreamBlob.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.IO;
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    public class StreamBlob : ByteArrayBlob
```

Comment:
>StreamBlob [](http://example.com/codeflow?start=17&length=10)

We should add to the comment that this will consume the entire stream (and not dispose it).